client/driver.go

```diff
@@ -97,6 +103,18 @@ type ResourceMeta struct {
 	AdvancePayment uint64
 }
 
+func defaultInit() (err error) {
```

Comment:
Why not use DSN parameters to pass config file path and master key values?

Reply:
DSN is used for connecting CovenantSQL database by many users. It should not contain local file path that probably not found in other machine.